src/lib/common/sol-platform-impl-systemd.c

```diff
@@ -53,9 +53,12 @@ struct ctx {
     struct sol_bus_client *timedate;
     struct sol_bus_client *hostname;
     struct sol_ptr_vector services;
+    bool locale_monitor_registered : 1;
+    bool timedate_monitor_registered : 1;
+    bool hostname_monitor_registered : 1;
 };
 
-static struct ctx _ctx;
+static struct ctx _ctx = { 0 };
```

Comment:
This explicit initialization is unneeded, the `static` guarantees that it will be zero initialized.